src/connectionController.ts

```diff
@@ -23,7 +24,11 @@ function getConnectWebviewContent() {
   </html>`;
 }
 
-export default class ConnectionManager {
+export enum DataServiceEventTypes {
+  connectionsDidChange
+}
+
+export default class ConnectionController {
```

Comment:
If you extend from require(‘events’).EventEmitter, you’ll get all of the listener/emit boilerplate for free.

Reply:
Good stuff - updated to use it.